components/event-publisher/org.wso2.carbon.event.publisher.core/src/main/java/org/wso2/carbon/event/publisher/core/internal/EventPublisher.java

```diff
@@ -280,4 +309,21 @@ public void process(Event event) {
     public org.wso2.siddhi.query.api.definition.StreamDefinition getStreamDefinition() {
         return streamDefinition;
     }
+
+    public class EventWrapper{
```

Comment:
Thanks @prabathabey, the requested change is already done in commit - https://github.com/wso2/carbon-analytics-common/commit/2bad33daa275fbf0bcd591c207e2ef67f5e454e0